frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx

```diff
@@ -7,14 +7,14 @@ import styles from "../../../Home/components/Motive/motive.module.scss";
 import "./custom-owl-carousel-style.scss";
 import { Modals } from "./Modal/index.js";
 import { END_POINT } from "./../../../../config/api";
-
+import LoaderFull from "../../../../components/util/Loader";
```

Comment:
Can we please a meaningful name maybe `Loader`

Reply:
Yes, Done.